cfgov/ask_cfpb/tests/test_search.py

```diff
@@ -319,14 +319,14 @@ def test_en_search_suggestion_es7(self, mock_search):
         mock_es_queryset = mock.Mock()
         mock_es_queryset.__iter__ = mock.Mock(return_value=iter([mock_return]))
         mock_es_queryset.count.return_value = 1
-        mock_search().query().highlight().filter().sort() \
+        mock_search().query().filter().sort() \
             .__getitem__().execute.return_value = [mock_return]
         mock_count = mock.Mock(return_value=1)
-        mock_search().query().highlight().count = mock_count
-        mock_search().query().highlight().filter().sort() \
+        mock_search().query().count = mock_count
+        mock_search().query().filter().sort() \
             .__getitem__().count = mock_count
```

Comment:
This will now fit on one line:

```suggestion
        mock_search().query().filter().sort().__getitem__().count = mock_count
```

Reply:
Done